app/views/microposts/create.js.erb

```diff
@@ -1,3 +1,6 @@
-alert("<%='Hurray Blog is created..'%>");
-var newblogs = document.querySelector("#blog-list");
-newblogs.insertAdjacentHTML("beforeend", "<%= j render 'micropost', micropost: @micropost %>");
+$.ajax({ success: function(result){
```

Comment:
@HrisikeshSinghaChowdhury why are you defining all these statements inside this ajax function? 
$.ajax is used for manually sending ajax request and handle the response.

Reply:
on getting successful ajax response i hide the add new section ....so i used it

Reply:
https://www.w3schools.com/jquery/ajax_ajax.asp

$.ajax({url: "demo_test.txt", success: function(result){
    $("#div1").html(result);
  }});

This is the syntax and this is used when manually sending the ajax request for that given url. For your case this is not the case. we are not sending the ajax through jquery ajax function, it is handled by rails through data-remote option.

Reply:
Ok I understood.....